style/style.css

```diff
@@ -632,3 +727,449 @@ form .line {
+
+  .title1 {
+    width: 920px;
+    height: 122px;
+    margin-left: 120px;
+    padding-top: 170px;
+  }
+
+  .info .title1,
+  .presentation1,
+  .connect1,
+  .social-icons1 {
+    margin-left: -15%;
+  }
+
+  .title1 h1 {
+    width: 500px;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .presentation1 {
+    width: 1440px;
+    height: 916px;
+  }
+
+  .presentation1 .intro1 {
+    margin-top: 136px;
+    width: 920px;
+    height: 98px;
+    margin-left: 15px;
+    font-size: 20px;
+  }
+
+  .social-icons1 {
+    height: 20px;
+    display: flex;
+    margin: 5.5rem 0;
+    align-items: center;
+    width: 11rem;
+    gap: 10px;
+    position: relative;
+    top: -124px;
+    list-style: none;
+  }
+
+  .connect1 .social-icons1 {
+    position: relative;
+    left: -59%;
+  }
+
+  .allcards {
+    width: 100%;
+    position: relative;
+    background-color: hsl(0, 0%, 90%);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-left: -40px;
+    padding: 142px 0;
+  }
+
+  .snapshoot1 {
+    display: none;
+  }
+
+  .snapshoot2 {
+    margin-top: -68px;
+    width: 50%;
+    padding: 10px;
+    display: flex;
+    height: 100%;
+  }
+
+  .left-block {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    height: 100%;
+    left: 0;
+  }
+
+  .card1,
+  .card3 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 46px;
+    width: 70%;
+  }
+
+  .card2,
+  .card4 {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    gap: 46px;
+    width: 70%;
+  }
+
+  .tonic-title-frame2 .break {
+    display: none;
+  }
+
+  .tags .ruby {
+    display: flex;
+  }
+
+  .text-desc p {
+    font-size: 16px;
+    float: left;
+    margin-top: 45px;
+    margin-left: 0;
+    width: 90%;
+    padding: 0;
+  }
+
+  .tonic-ul {
+    margin-top: 20px;
+    margin-bottom: 36px;
+    padding: 0;
+    float: left;
+    width: 100%;
+    height: auto;
+  }
+
+  ul.tonic-ul {
+    margin: 0;
+    margin-left: -20px;
+  }
+
+  .tonic-ul li {
+    font-size: 18px;
+    font-weight: 400;
+    width: auto;
+  }
+
+  .tonic-title-frame1 {
+    width: 100%;
+    height: auto;
+  }
+
+  .tonic-title-frame1 h2 {
+    font-size: 40px;
+    font-weight: 700;
+    margin: 0;
+  }
+
+  .tonic-title-frame2 h2 {
+    font-size: 40px;
+    font-weight: 700;
+    margin: 0;
+  }
+
+  .tags ul {
+    float: left;
+    margin: 0;
+    margin-left: -15px;
+    width: 100%;
+    height: auto;
+  }
+
+  .tags ul .tag-pl {
+    width: auto;
+    padding: 4px 12px;
+    margin-top: -20px;
+    line-height: 12px;
+  }
+
+  .btn button {
+    margin: 0;
+  }
+
+  .btn {
+    margin-left: 25px;
+    margin-top: -15px;
+  }
+
+  /* About myself */
+  .about {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-bottom: 130px;
+    padding-top: 140px;
+    width: 90%;
+  }
```

Comment:
- I think your `about` section is a bit too left-aligned as well, can you make it a bit more centered? 🤗 

### Your about section 👇🏽 
![abtt](https://user-images.githubusercontent.com/99246359/200012786-6a5bc137-6c23-4a22-9227-5837260af113.PNG)

### Figma

![figabt](https://user-images.githubusercontent.com/99246359/200013493-262d237d-a63b-49df-b34b-aedc644bd6bc.PNG)

Reply:
@Favourezeugwa thank you so much :relaxed: for your accurate review, I am going to applicate those changes.